packages/typespec-ts/test/integration/lroCore.spec.ts

```diff
@@ -42,11 +42,7 @@ describe("AzureLroCoreClient Rest Client", () => {
     try {
       const initalResponse = await client
         .path("/azure/core/lro/standard/users/{name}", "madge")
-        .delete({
-          body: {
-            role: "contributor"
-          }
-        });
```

Comment:
why we have changes on the RLC lro test ?

Reply:
There is no need to pass the body for this delete, just correct this test case.